cmd/plasmacli/store/keystore.go

```diff
@@ -214,6 +217,76 @@ func ImportECDSA(name string, pk *ecdsa.PrivateKey) (ethcmn.Address, error) {
 
 }
 
+func Import(name string, keyJSON []byte) (ethcmn.Address, error) {
+	dir := getDir(accountsDir)
+	db, err := leveldb.OpenFile(dir, nil)
+	if err != nil {
+		return ethcmn.Address{}, fmt.Errorf("leveldb: %s", err)
+	}
+	defer db.Close()
+
+	buf := cosmoscli.BufferStdin()
+	pass, err := cosmoscli.GetPassword(ImportPassphrasePrompt, buf)
+	if err != nil {
+		return ethcmn.Address{}, fmt.Errorf("error with imported password: %s", err)
+	}
+	newpass, err := cosmoscli.GetCheckPassword(NewPassphrasePrompt, NewPassphrasePromptRepeat, buf)
+	if err != nil {
+		return ethcmn.Address{}, fmt.Errorf("error with new password: %s", err)
+	}
+
+	acc, err := ks.Import(keyJSON, pass, newpass)
+	if err != nil {
+		return ethcmn.Address{}, fmt.Errorf("error importing keyfile: %s", err)
+	}
+
+	key := []byte(name)
+	if err = db.Put(key, acc.Address.Bytes(), nil); err != nil {
+		return ethcmn.Address{}, fmt.Errorf("leveldb: %s", err)
+	}
+
+	return acc.Address, nil
+}
+
+// Export exports as a JSON key, encrypted with newPassphrase
+func Export(name string) (keyJSON []byte, err error) {
+	// Get Account from Keystore
+	dir := getDir(accountsDir)
+	db, err := leveldb.OpenFile(dir, nil)
+	if err != nil {
+		return nil, fmt.Errorf("leveldb: %s", err)
+	}
+	defer db.Close()
+
+	key := []byte(name)
+	addr, err := db.Get(key, nil)
+	if err == leveldb.ErrNotFound {
+		return nil, fmt.Errorf("account does not exist")
+	} else if err != nil {
+		return nil, fmt.Errorf("leveldb: %s", err)
+	}
+
+	acc := accounts.Account{
+		Address: ethcmn.BytesToAddress(addr),
+	}
+
+	buf := cosmoscli.BufferStdin()
+	pass, err := cosmoscli.GetPassword(PassphrasePrompt, buf)
```

Comment:
ffti (feel free to ignore): It would be nice to failfast here if the password is incorrect.

Reply:
Will ignore for now as there aren't great ways to check if a password is valid atm. Might post an issue on implementing a method to fail fast on password failures

Reply:
👌

Reply:
👌

Reply:
👌

Reply:
👌